src/client/src/OpenFin/Snapshots/Snapshots.tsx

```diff
@@ -38,11 +37,11 @@ const useSnapshots = () => {
 }
 
 interface ListProps {
-  snapshots: Record<string, Snapshot>
+  snapshots: Record<string, OpenFin.Snapshot>
```

Comment:
may have missed this, but where does the global OpenFin come from .. is that related to package version updates?

Reply:
it's a namespace declared in `openfin-adapter`